### DISCUSSION TITLE
SHA256 checksum does not match

### DISCUSSION THREAD

**Discussion by orangeduu** [openai/whisper#920] [Q&A] [answered]
Hello! This is my first time trying anything as complicated as this ai program, and I've been running into some trouble. When trying to translate using the described method in the repository, I get the following warning
`Warning: C:\Users\eango\.cache\whisper\small.pt exists, but the SHA256 checksum does not match; re-downloading the file
  warnings.warn(f"{download_target} exists, but the SHA256 checksum does not match; re-downloading the file")`
afterwards It begins to work, but then gives me another warning of:
`C:\Python Software\New folder\lib\site-packages\whisper\transcribe.py:78: UserWarning: FP16 is not supported on CPU; using FP32 instead
  warnings.warn("FP16 is not supported on CPU; using FP32 instead")`
and then starts to translate.

This only works if it's a short clip. When I tried with a larger, 18 minute clip, it wouldnt even finish processing.  Is there a way I could better do this? Apologies if this is a dumb question

**Comment by jongwook** [comment, marked as answer]
Hi! The first warning may happen if the model download was aborted during the previous try, and the second warning just means the calculation will happen in a slower but higher-precision format. They are both safe to ignore.

My guess is that because you're not using a GPU, it's just taking too long to transcribe the audio, especially when there are lots of words spoken in the audio. If you specify `--model tiny.en`, it will run faster than the default `small` model that you're using.

If you don't currently have a GPU, you can try the example notebooks like [this one](https://colab.research.google.com/github/openai/whisper/blob/master/notebooks/LibriSpeech.ipynb) on Google Colab which provides a GPU runtime.

**Reply by orangeduu** [reply]
Thanks so much! that notebook really helped!

**Comment by xiaofanloveme** [comment]
Hello! I have the similar problem as SHA256 checksum does not match, but I have the different situation. The detail is that when I try to translate an audio, the progress bar will automatically stop at about 70%, and show “ConnectionResetError: [WinError 10054] The remote host forced the closure of an existing connection”. Is there a way I could solve this problem?

**Reply by SAR-hub** [reply]
Same query. Did you find any solution?

**Reply by xiaofanloveme** [reply]
Yes, I have solve the problem. you should firstly verify ffmpeg. I use 'git clone https://git.ffmpeg.org/ffmpeg.git ffmpeg' to install ffmpeg. In addition, Open Anaconda Powershell Prompt as an administrator
